db/schema.rb

```diff
@@ -36,6 +36,7 @@
     t.integer  "park_id",    null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer  "rating"
```

Comment:
Remove this attribute, you're going to have to implement a rating/voting system that will require additional tables